decentraland/unity-explorer
# Destroy out of range scene entities

## What does this PR change?



This change reduces memory and CPU usage of PartitionSceneEntitiesSystem and ResolveVisualSceneStateSystem.

fix: #2694

## How to test the changes?



1. Launch the explorer
2. Walk around the Genesis City a lot, look at the profiler, and observe that the game uses a couple megabytes less memory than without this change and that CPU usage of ResolveVisualSceneStateSystem stays low.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
